exercises/sum.cpp

```diff
@@ -6,3 +6,26 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+template <typename T>
+T sum (T n1, T n2){
+  return n1+n2;
+}
+
+int main(){
+
+  double n1,n2;
+  cout << "Inserire il primo numero:";
+  cin >> n1;
+
+  cout << "Inserire il secondo numero:";
+  cin >> n2;
+
+  cout << "La somma fra i due numeri e': " << sum<double>(n1,n2);
+
+  return 0;
```

Comment:
Meglio usare una macro per i valori handcoded (non una variabile perchè occupa memora, il simbolo di macro viene sostituito con il valore reale in fase di compilazione, per info: https://www.arduino.cc/reference/en/language/structure/further-syntax/define/):
```diff
+ #define EXIT_SUCCESS 0
+  return EXIT_SUCCESS;
- return 0;
```